src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMissingLeadingAsteriskCheck.java

```diff
@@ -0,0 +1,209 @@
+ * </p>
+ * <ul>
+ * <li>
+ * Property {@code violateExecutionOnNonTightHtml} - Control when to print violations if the
+ * Javadoc being examined by this check violates the tight html rules defined at
+ * <a href="https://checkstyle.org/writingjavadocchecks.html#Tight-HTML_rules">Tight-HTML Rules</a>.
+ * Type is {@code boolean}. Default value is {@code false}.
+ * </li>
+ * </ul>
+ * <p>
+ * To configure the check:
+ * </p>
+ * <pre>
+ * &lt;module name="JavadocMissingLeadingAsterisk"/&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * &#47;**
+ *  * Valid Java-style comment.
+ *  *
+ *  * &lt;pre&gt;
+ *  *   int value = 0;
+ *  * &lt;/pre&gt;
+ *  *&#47;
+ * class JavaStyle {} // ok
+ *
+ * &#47;** Valid Scala-style comment.
+ *   * Some description here.
+ *   **&#47;
+ * class ScalaStyle {} // ok
+ *
+ * &#47;** **
+ *  * Asterisks on first and last lines are optional.
+ *  * *&#47;
+ * class Asterisks {} // ok
+ *
+ * &#47;** No asterisks are required for single-line comments. *&#47;
+ * class SingleLine {} // ok
+ *
+ * &#47;** // violation on next blank line, javadoc has lines without leading asterisk.
+ *
+ *  *&#47;
+ * class BlankLine {}
+ *
+ * &#47;** Wrapped
+ *     single-line comment *&#47; // violation, javadoc has lines without leading asterisk.
+ * class Wrapped {}
+ *
+ * &#47;**
+ *  * &lt;pre&gt;
+ *     int value; // violation, javadoc has lines without leading asterisk.
+ *  * &lt;/pre&gt;
+ *  *&#47;
+ * class Code {}
+ * </pre>
+ * <p>
+ * Parent is {@code com.puppycrawl.tools.checkstyle.TreeWalker}
+ * </p>
+ * <p>
+ * Violation Message Keys:
+ * </p>
+ * <ul>
+ * <li>
+ * {@code javadoc.missed.html.close}
+ * </li>
+ * <li>
+ * {@code javadoc.missing.asterisk}
+ * </li>
+ * <li>
+ * {@code javadoc.parse.rule.error}
+ * </li>
+ * <li>
+ * {@code javadoc.wrong.singleton.html.tag}
+ * </li>
+ * </ul>
+ *
+ * @since 8.38
+ */
+@StatelessCheck
+public class JavadocMissingLeadingAsteriskCheck extends AbstractJavadocCheck {
+
+    /**
+     * A key is pointing to the warning message text in "messages.properties"
+     * file.
+     */
+    public static final String MSG_MISSING_ASTERISK = "javadoc.missing.asterisk";
+
+    @Override
+    public int[] getRequiredJavadocTokens() {
+        return new int[] {
+            JavadocTokenTypes.NEWLINE,
+        };
+    }
+
+    @Override
+    public int[] getAcceptableJavadocTokens() {
+        return getRequiredJavadocTokens();
+    }
+
+    @Override
+    public int[] getDefaultJavadocTokens() {
+        return getRequiredJavadocTokens();
+    }
+
+    @Override
+    public void visitJavadocToken(DetailNode detailNode) {
+        DetailNode nextSibling = getNextNode(detailNode);
+
+        // Till https://github.com/checkstyle/checkstyle/issues/9005
+        // Due to bug in the Javadoc parser there may be phantom description nodes.
+        while (nextSibling.getType() == JavadocTokenTypes.DESCRIPTION
+                || nextSibling.getType() == JavadocTokenTypes.WS) {
```

Comment:
Please change to TokenUtil.isOfType method

Reply:
done